src/Microsoft.VisualBasic/ref/Microsoft.VisualBasic.cs

```diff
@@ -198,11 +198,8 @@ public sealed partial class OptionTextAttribute : System.Attribute
     public sealed partial class ProjectData
     {
         internal ProjectData() { }
-        [System.Security.SecuritySafeCriticalAttribute]
```

Comment:
Yes, but those only matter at runtime. They do not matter at compile time where the contracts are consumed.